superset-frontend/src/components/Chart/ChartRenderer.jsx

```diff
@@ -90,10 +88,7 @@ class ChartRenderer extends Component {
     )?.suppressContextMenu;
     this.state = {
       showContextMenu:
-        props.source === ChartSource.Dashboard &&
-        !suppressContextMenu &&
-        (isFeatureEnabled(FeatureFlag.DrillToDetail) ||
```

Comment:
I think we should keep the check for drill to detail here

Reply:
+1